test/cmocka/src/math/arithmetic/exponential.c

```diff
@@ -68,7 +69,11 @@ static void test_math_arithmetic_exponential_fixed(void **state)
 	for (i = 0; i < 256; i++) {
 		gen_exp_testvector(a_tmp, b_tmp, &r, &b_i);
 		a_i = (double)b_i / (1 << 28);
+#if (EXPONENTIAL_HIFI4 == 1)
```

Comment:
EXPONENTIAL_HIFI4, suggest this one longer, I remember there is another exp module, we need different with it.
also, does this hifi only works for HIFI4 dsp? if also works for HIFI3, please add HIFI3, it is an optimization, no need use EXPONENTIAL_HIFI4, how about just use:
XCHAL_HAVE_HIFI3 == 1 || XCHAL_HAVE_HIFI4 == 1

Reply:
Ahaah, Please give it a little more time. I 'm in the  middle of resolving the merge conflict. I will notify once I am done. 
Thank you

Reply:
@btian1 , I've made my next commit, and you can review the HIFi3 improvements on top of the HiFi4 and HiFi5 versions .